### DISCUSSION TITLE
Crash in MediaPlayerCS sample application

### DISCUSSION THREAD

**Discussion by kuttipoo** [ffmpeginteropx/FFmpegInteropX#359] [Q&A]
I have updated the 'MediaPlayerCS' sample app to utilize the stable release NuGet packages of 'FFmpegInteropX' and 'FFmpegInteropX.FFmpegUWP', and I built it using the 'Debug - x64' configuration (Visual studio 2019 & Windows 11). However, the app crashes at the following line of code, where I attempt to instantiate the 'FFmpegMediaSource' using the provided URI:

FFmpegMSS = await FFmpegMediaSource.CreateFromUriAsync(uri, Config); (mainPage.xaml.cs, line 302)
The error message I am encountering is:

"Exception thrown at 0x00007FFAED9C1F70 (FFmpegInteropX.dll) in MediaPlayerCS.exe: 0xC0000005: Access violation writing location 0x0000000000000028."

For your reference, I have attached the corresponding call stack file.
[ffnpegInteropXCallStack.txt](https://github.com/ffmpeginteropx/FFmpegInteropX/files/12306898/ffnpegInteropXCallStack.txt)

Thanks for the great project, and any help with this issue would be greatly appreciated.

-Shanky

**Comment by brabebhin** [comment]
I will take a look at it.
However the samples are mostly supposed to go in sync with the current version of the code in the sln with direct reference. The stable nuget had some bugs, and you should use the latest pre-release, which have since fixed them. We didn't have time to bump up versions of stable, and we will probably wait until some of the outstanding PRs are merged.

**Reply by kuttipoo** [reply]
Thanks for the quick response. It is a user error; the RTSP URI I'm trying out requires authentication. If the credentials are not correct, the 'FFmpegInteropX' throws an exception. Thanks for the suggestion to try out the latest pre-release; it seems more stable with streaming.
